js/tests/unit/collapse.spec.js

```diff
@@ -900,41 +902,49 @@ describe('Collapse', () => {
         const target2 = fixtureEl.querySelector('#test2')
 
         const target2Shown = () => {
-          expect(trigger1).not.toHaveClass('collapsed')
-          expect(trigger1.getAttribute('aria-expanded')).toEqual('true')
-
-          expect(trigger2).not.toHaveClass('collapsed')
-          expect(trigger2.getAttribute('aria-expanded')).toEqual('true')
-
-          expect(trigger3).not.toHaveClass('collapsed')
-          expect(trigger3.getAttribute('aria-expanded')).toEqual('true')
-
-          target2.addEventListener('hidden.bs.collapse', () => {
+          setTimeout(() => {
             expect(trigger1).not.toHaveClass('collapsed')
             expect(trigger1.getAttribute('aria-expanded')).toEqual('true')
 
-            expect(trigger2).toHaveClass('collapsed')
-            expect(trigger2.getAttribute('aria-expanded')).toEqual('false')
+            expect(trigger2).not.toHaveClass('collapsed')
+            expect(trigger2.getAttribute('aria-expanded')).toEqual('true')
 
             expect(trigger3).not.toHaveClass('collapsed')
             expect(trigger3.getAttribute('aria-expanded')).toEqual('true')
 
-            target1.addEventListener('hidden.bs.collapse', () => {
-              expect(trigger1).toHaveClass('collapsed')
-              expect(trigger1.getAttribute('aria-expanded')).toEqual('false')
+            target2.addEventListener('hidden.bs.collapse', () => {
+              setTimeout(() => {
+                expect(trigger1).not.toHaveClass('collapsed')
+                expect(trigger1.getAttribute('aria-expanded')).toEqual('true')
 
-              expect(trigger2).toHaveClass('collapsed')
-              expect(trigger2.getAttribute('aria-expanded')).toEqual('false')
+                expect(trigger2).toHaveClass('collapsed')
```

Comment:
Blank character at the beginning of the line

Reply:
IMO, they're well aligned on the final version of this file, if not, feel free to unresolve this comment.